index.js

```diff
@@ -39,7 +39,7 @@ function Color(obj, model) {
 	var i;
 	var channels;
 
-	if (!obj) {
+	if (typeof obj === 'undefined') {
```

Comment:
@Qix-  I changed this condition to allow empty string to be deal by color-string. However, I am not sure if there are conditions other than undefined that need to be dealt here.

Reply:
Aha yes. I see now, this is correct.